test/unit/resources/aws_security_group_test.rb

```diff
@@ -320,6 +340,114 @@ def describe_security_groups(query)
             }),
           ],
           ip_permissions_egress: [],
+        }),
+        OpenStruct.new({
+          description: 'Open for group one rule',
+          group_id: 'sg-33334441',
+          group_name: 'zeta',
+          vpc_id: 'vpc-12345678',
+          ip_permissions: [
+            OpenStruct.new({
+              from_port: nil,
+              to_port: nil,
+              ip_protocol: "-1",
+              user_id_group_pairs: [
+                OpenStruct.new({
+                  description: "String",
```

Comment:
I think this is confusing - if you are not using or testing a field, it is better to omit it.  If the value 'String' is present, that might imply to someone that the value is important, or that we are setting up type checking - when in fact it isn't.

Reply:
That's a good example of copy-paste programming. I copied that stuff from the API docs vanilla example. Forgot to change the content after that. I've removed the strings optional according to the documentation. The remaining have sensible default content now.